### DISCUSSION TITLE
How do I update a local instance of Monkeytype?

### DISCUSSION THREAD

**Discussion by Ottaviocr** [monkeytypegame/monkeytype#4131] [Questions]
I run a local instance of MT (frontend only) on Linux following https://github.com/monkeytypegame/monkeytype/blob/master/CONTRIBUTING_ADVANCED.md#building-and-running-monkeytype 

If I run:

`$ git pull -r`

from the monkeytype directory, do I have to rerun `npm run install-all` in the project root again?

**Comment by EMRD95** [comment]
It updates itself automatically for front end changes, you could check the logs live.

**Reply by Ottaviocr** [reply]
Do you mean, it will update without "git pull -r"?
